roles/health_checks/README.md

```diff
@@ -0,0 +1,53 @@
+# Health Checks
+The role enables user to manage the ACL resources independent of platforms and perform ACL health checks.
+
+## Capabilities
+
+**ACL Health Checks** :
+- Enables users to perform health checks for ACL configurations and their interface assignments.
+- Users can perform the following health checks:
+  - `available_acls`: Lists all ACLs configured on the device.
+  - `missing_acls`: Identifies ACLs referenced in interfaces but not defined.
+  - `unassigned_acls`: Shows ACLs that are defined but not used on any interface.
+  - `interface_acl_status`: Shows the status of ACL assignments on interfaces.
+
+This role enables users to create a runtime brownfield inventory with all the ACLS configurations in terms of host vars. These host vars are ansible facts that have been gathered through the *acl_interfaces and *acls network resource module. The tasks offered by this role can be observed below:
```

Comment:
I believe it needs to be corrected w.r.t health-checks

Reply:
yes this line tells the functionality related to the persist role not the heath_check one.